internal/controllers/node_label_module_version_reconciler.go

```diff
@@ -141,6 +144,18 @@ func (nlmvha *nodeLabelModuleVersionHelper) getDevicePluginPods(ctx context.Cont
 	return devicePluginPods, nil
 }
 
+func (nlmvha *nodeLabelModuleVersionHelper) getLoadedKernelModules(node v1.Node) []types.NamespacedName {
```

Comment:
How about replacing the `v1.Node` parameter with a `map[string]string` (the labels) directly?
Also, I don't think this should be a method.

Reply:
changed the v1.Node to map[string]string
Regarding method -  we know that we don't agree on that :). Lets leave that discussion for the refactoring task that we scheduled for v2.1